libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -427,11 +427,7 @@ void AP_MotorsHeli_Dual::move_actuators(float roll_out, float pitch_out, float c
     }
 
     // updates below land min collective flag
-    if (collective_out <= _collective_land_min_pct) {
-        _heliflags.below_land_min_coll = true;
-    } else {
-        _heliflags.below_land_min_coll = false;
-    }
+    _heliflags.below_land_min_coll = !is_positive(collective_out - _collective_land_min_pct);
```

Comment:
I don't understand what this fixes?   It seems like what was there and what you are proposing are the same thing.